app/src/main/java/com/limelight/binding/input/trackpad/TrackpadHandler.java

```diff
@@ -0,0 +1,112 @@
+package com.limelight.binding.input.trackpad;
+
+import android.view.KeyEvent;
+import android.view.MotionEvent;
+
+import com.limelight.binding.input.KeyboardTranslator;
+import com.limelight.binding.input.trackpad.Core.TrackpadFramework;
+import com.limelight.binding.input.trackpad.Core.TrackpadListener;
+import com.limelight.nvstream.NvConnection;
+import com.limelight.nvstream.input.KeyboardPacket;
+import com.limelight.nvstream.input.MouseButtonPacket;
+
+public class TrackpadHandler implements TrackpadListener {
+
+    private final KeyboardTranslator keyboardTranslator = new KeyboardTranslator();
+
+    private final NvConnection conn;
+    private final float sense;
+
+    // Bring the Core Class
+    private final TrackpadFramework framework;
+
+    // You should call it from a class that requires trackpad input.
+    public TrackpadHandler(NvConnection conn,float sense){
+        framework = new TrackpadFramework(this);
+        this.conn = conn;
+        this.sense = sense;
```

Comment:
What is `sense` used for? I see it's always set to 1.

Reply:
Ready for future features.

I haven't created a trackpad sensitivity setting yet.

When the trackpad sensitivity setting is created, we plan to receive the sense value from prefConfig and apply the sensitivity when creating "TrackpadHandler" in "Game.java".

So, since I didn't make a setting at the moment, I fixed it to 1.